sys/meson_git_wrapper.py

```diff
@@ -25,9 +25,7 @@ def isCArgSupported(executable, path):
 
 def simple_git_execution(args):
     try:
-        called = subprocess.run(args, check=True, capture_output=True)
-        with open("gittip", "w", encoding="utf8") as f:
-            f.write(called.stdout.decode("utf8").strip())
+        called = subprocess.run(args, check=True)
```

Comment:
This is wrong, and using `capture: true` in Meson is bad -- it causes this python script to be run by `meson --internal exe`, which imports lots of stuff and is very slow and obscures the command line.

You don't need to throw the baby out with the bathwater. capture_output is python 3.7, sure, but it's also literally an alias for `stdout=subprocess.PIPE, stderr=subprocess.PIPE` and it's extremely easy to make that work with python 3.6.

Reply:
Thanks a lot for this hint!

Reply:
Great. Happy to help. :)